src/Microsoft.AspNetCore.JsonPatch/Resources.resx

```diff
@@ -142,7 +142,7 @@
     <value>The path segment '{0}' is invalid for an array index.</value>
   </data>
   <data name="InvalidJsonPatchDocument" xml:space="preserve">
-    <value>The type '{0}' was malformed and could not be parsed.</value>
+    <value>The JsonPatchDocument was malformed and could not be parsed.</value>
```

Comment:
`The JSON patch document was malformed and could not be parsed.`

The error message refers to the input, not to the specific type.